pkg/controller/volume/expand/expand_controller.go

```diff
@@ -339,3 +340,8 @@ func (expc *expandController) GetSubpather() subpath.Interface {
 	// not needed for expand controller
 	return nil
 }
+
+func (expc *expandController) GetCSIDriverLister() storagelisters.CSIDriverLister {
```

Comment:
this can be removed if you refactor this function into the ADC and Kubelet specific interfaces

Reply:
Done